internal/emlb/emlb.go

```diff
@@ -233,6 +233,72 @@ func (e *EMLB) ReconcileVIPOrigin(ctx context.Context, machineScope *scope.Machi
 	return nil
 }
 
+// DeleteLoadBalancer deletes the Equinix Metal Load Balancer associated with a given ClusterScope.
+func (e *EMLB) DeleteLoadBalancer(ctx context.Context, clusterScope *scope.ClusterScope) error {
+	log := ctrl.LoggerFrom(ctx)
+
+	packetCluster := clusterScope.PacketCluster
+	clusterName := packetCluster.Name
+
+	// Make sure the cluster already has an EMLB ID in its packetCluster annotations, otherwise abort.
+	lbID, exists := packetCluster.Annotations[loadBalancerIDAnnotation]
+	if !exists || (lbID == "") {
+		log.Info("no Equinix Metal Load Balancer found in cluster's annotations, skipping EMLB delete")
+		return nil
+	}
+
+	log.Info("Deleting EMLB", "Cluster Metro", e.metro, "Cluster Name", clusterName, "Project ID", e.projectID, "Load Balancer ID", lbID)
+
+	// Fetch the Load Balancer object.
+	// Skip if 404, otherwise error
```

Comment:
Looks like we are not skipping

Reply:
seems we'll need getLoadBalancer to return the resp object so we can detect the 404

Reply:
> [refactor: take marques suggestion to remove unnecessary get calls](https://github.com/kubernetes-sigs/cluster-api-provider-packet/pull/771/commits/70b4b26fddc0782bb0bf1b0ad1a1f6397a00d5d0)

I wonder where you got that idea..

> seems we'll need getLoadBalancer to return the resp object so we can detect the 404

😄